libs/documentation/src/lib/storybook/actions/actions.stories.ts

```diff
@@ -59,6 +61,15 @@ export default {
         category: 'ActionMenuModel/trigger',
       },
     },
+    label: {
+      control: {
+        type: 'text',
+      },
+
+      table: {
+        category: 'ActionMenuModel/actions',
```

Comment:
Since label isnt part of the actions object, the category should be 'ActionMenuModel'

Reply:
Updated the catagory